src/content/docs/ru/tutorial/4-layouts/2.mdx

```diff
@@ -0,0 +1,180 @@
+---
+<h1> {frontmatter.title}</h1>
+<p>{frontmatter.pubDate.slice(0,10)}</p>
+<p><em>{frontmatter.description}</em></p>
+<p>Написал: {frontmatter.author}</p>
+<img src={frontmatter.image.url} width="300" alt={frontmatter.image.alt} />
+<slot />
+```
+</Box>
+
+:::note[avoid duplication]
+ Все, что отображается вашим макетом, **не нужно** вводить в ваш пост в блоге! Если вы заметите какое-либо дублирование, когда проверяете предварительный просмотр в браузере, обязательно удалите контент из файла Markdown.
+ :::
+
+
+
+<Box icon="question-mark">
+
+### Проверьте свои знания
+Вы можете разобраться, что должно быть в пробелах, чтобы следующие два компонента вместе создали рабочий код Astro?
+
+1.  ```markdown title="src/pages/posts/learning-astro.md"
+    ---
+    layout: ../../__________/MyMarkdownLayout.astro
+    title: "Learning About Markdown in Astro"
+    author: Astro Learner
+    ____: 2022-08-08
+    ---
+    Я так много узнал сегодня! Astro позволяет мне писать на Markdown, но также использовать переменные из frontmatter. Я даже могу получить доступ к этим значениям в компоненте макета Astro.
+    ```
+
+2.  ```astro title="src/layouts/MarkdownLayout.astro"
+    ---
+    import ____________ from '../components/Footer.astro'
+    const { ___________ } = Astro.props
+    ---
+    <h1>{frontmatter.title}</h1>
+    <p>Написал:{frontmatter.______ } on {frontmatter.pubDate}</p>
+    < _______ />
+    <Footer />
+    ```
+    <details>
+        <summary>Покажите заполненные пробелы!</summary>
+
+        1.  ```markdown title="src/pages/posts/learning-astro.md" "../layouts" "pubDate:"
+            ---
+            layout: ../../layouts/MyMarkdownLayout.astro
+            title: "Learning About Markdown in Astro"
+            author: Astro Learner
+            pubDate: 2022-08-08
+            ---
+            I learned so much today! Astro allows me to write in Markdown, but also use variables from the frontmatter. I can even access those values in an Astro layout component.
+            ```
+
+        2.  ```astro title="src/layouts/MarkdownLayout.astro" " Footer " " frontmatter " ".author" "slot"
+            ---
+            import Footer from '../components/Footer.astro'
+            const { frontmatter } = Astro.props
+            ---
+            <h1>{frontmatter.title}</h1>
+            <p>Написал:{frontmatter.author } on {frontmatter.pubDate}</p>
+            < slot />
+            <Footer />
+            ```
+    </details>
+</Box>
+
+<Box icon="check-list">
+
+## Чек-лист
+
+<Checklist>
+- [ ] Я могу добавить свойство макета в запись в блоге в ее YAML frontmatter.
+- [ ] Я могу создавать отдельный макет для записей в блоге с помощью Markdown.
+- [ ] Я могу использовать значения из frontmatter записей блога в компоненте макета.
+</Checklist>
+
+</Box>
+
+### Ресурсы
+
+- [Markdown Layouts(макеты) в Astro](/ru/guides/markdown-content/#frontmatter-layout)
+
+- [Markdown Layout Props](/ru/core-concepts/layouts/#markdown-layout-props)
```

Comment:
```suggestion
{/* This link should EVENTUALLY go directly to "/ru/core-concepts/layouts/#markdown-layout-props" once the Layouts page is updated and this section exists*/}
- [Markdown Layouts](/ru/core-concepts/layouts/#markdown-макеты)
```
The Russian version of the Layouts page is outdated, so this link should actually work (until the Russian page is updated!)